crates/holochain_cascade/src/authority.rs

```diff
@@ -77,6 +83,7 @@ pub async fn handle_get_agent_activity_deterministic(
     Ok(results)
 }
 
+/// Handler for get_links query to a Record/Entry authority
```

Comment:
```suggestion
/// Handler for get_links query to a Entry authority
```
It's still an entry authority even though the base can be any hash because the basis is found by turning any hash into an entry hash.

Reply:
@freesig but a link can be from an ActionHash, so wouldn't that be the record authority?